src/lib.rs

```diff
@@ -285,9 +293,14 @@ impl<R: gimli::Reader> Context<R> {
         Some(unit_id)
     }
 
+    /// Find the DWARF unit corresponding to the given virtual memory address.
+    pub fn find_dw_unit(&self, probe: u64) -> Option<Rc<gimli::Unit<R>>> {
```

Comment:
Please don't use shortened names for public APIs. (It's good to avoid them for internal APIs too, but there's already a lot of that.)
```suggestion
    pub fn find_dwarf_unit(&self, probe: u64) -> Option<Rc<gimli::Unit<R>>> {
```

Reply:
Done